### PR TITLE
[update] changes to remove IdP access token from profile

### PR DESCRIPTION
**Pull request by mcastany**
Docs to be included:

https://auth0.com/docs/migrations
https://auth0.com/docs/tokens/idp
https://auth0.com/docs/what-to-do-once-the-user-is-logged-in/calling-an-external-idp-api
https://auth0.com/docs/quickstart/native/xamarin
https://auth0.com/docs/user-profile/normalized
https://auth0.com/docs/quickstart/native/windows-uwp-javascript

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/what-to-do-once-the-user-is-logged-in/calling-an-external-idp-api.md [hunk 1]
```diff
@@ -12,21 +12,25 @@ This document details the recommended method for calling a third-party IdP witho
 
 The basic flow is the following:
 
-* Create a client that will be used to get an access\_token with the `read:user_idp_tokens` scope valid to obtain authorized access to the Auth0 Management API. 
-* Create a backend service to handle your request to the external IdP API. It will:  
-    1. Execute the client credentials exchange to get a valid APIv2 access token.
-    2. Use the access token to execute the request to the `/api/v2/users/{user_id}` endpoint.
-    3. Use the IdP access token extracted from `user.identities[0].access_token` to call the IdP API.
+* Create a client that will be used to get an access token with the `read:user_idp_tokens` scope valid to obtain authorized access to the Auth0 Management API.
```
Comment (Catografix) [inline articles/what-to-do-once-the-user-is-logged-in/calling-an-external-idp-api.md]: move these details to the following section

File: articles/what-to-do-once-the-user-is-logged-in/calling-an-external-idp-api.md [hunk 2]
```diff
@@ -60,15 +67,22 @@ Now with the Access token, call the `/api/v2/users/{user-id}` endpoint from the
...
-- `user-id`: The property user_id of the profile.
+The request must include an `Authorization` header with `Bearer bearer-token`, where `bearer-token` is the Auth0 token you obtained in the previous step. 
+
+The body of the request must include the following:
+
+- `user-id`: The value of the `user_id` property of the user profile.
+
+### 3. Extract the IdP access token
+
+Use the IdP access token extracted from `user.identities[0].access_token` to make calls to the IdP API.
```
Comment (Catografix) [inline articles/what-to-do-once-the-user-is-logged-in/calling-an-external-idp-api.md:96]: add details, i guess extract token and call IdP can be one step, but maybe better as two?

**Comment by Catografix** [comment]
Shortcut to doc: https://auth0-docs-content-pr-1718.herokuapp.com/docs/what-to-do-once-the-user-is-logged-in/calling-an-external-idp-api

**Pull request merged**